### ISSUE TITLE
auto, session, skins, api status is red

### ISSUE THREAD

**Issue by SpookyTabz** [gorilla-devs/GDLauncher#1137]
![wetrs](https://user-images.githubusercontent.com/45498671/139450960-41a1c700-a1cd-4988-a18c-f9b42d310285.PNG)

**Comment by Eskaan** [comment]
Yeah we know. Mojang took their session Servers down. Will fix it eventually but its not that urgent.

**Comment by SpookyTabz** [comment]
i cant download any modpacks

**Comment by SpookyTabz** [comment]
it just stops at downloading forge installer...

**Comment by Eskaan** [comment]
Yeah, upgrade to the latest beta (take a look at #1086 )

**Comment by SpookyTabz** [comment]
when i try to install v1.1.15-beta.6 i get this error
![sa](https://user-images.githubusercontent.com/45498671/139457969-6529072c-41e7-445f-9dcd-1e3f45d71953.png)
